mtbl.pyx

```diff
@@ -31,12 +46,15 @@ def varint_length(uint64_t value):
     """varint_length(v) -> number of bytes the integer v would require in varint encoding."""
     return mtbl_varint_length(value)
 
-def varint_length_packed(bytes py_buf):
+def varint_length_packed(py_buf):
```

Comment:
Type removed?

Reply:
I think bytes was removed to allow bytearrays as shown in the [test here](https://github.com/farsightsec/pymtbl/pull/10/commits/635564b61a7086334a79ab7eb803d80b13f57d23?file-filters%5B%5D=.py#diff-27be8287998ceb243b81448f80da9cec2e0f4b8f42ee5f260772333c5d237bd1R42)